### DISCUSSION TITLE
New Game Mode: Death Match

### DISCUSSION THREAD

**Discussion by mrdrprofuroboros** [BattlesnakeOfficial/feedback#163] [General]
There was a DOS game "Snake Battle" in my childhood of the same basic idea but the game rules were completely different:

TLDR; https://www.youtube.com/watch?v=t5NMuIA4WpI&ab_channel=Reidea

1. Movement:
* snakes don't die when they collide objects, they just get stuck until they have possibility to go on
* making zero movements on purpose is not allowed though
* snakes can eat each other if they bite tails (head steps a cell with others snake tail)
* biting snake elongates by 1, bited snake shortens by 1
* when 2-parts head-tail snake is bitten, it dies leaving a tail on the field (just static food)

2. Board layout:
* 25x25 field
* snakes start as 10-cell long straight sticks positioned from center of each side towards square center

3. Food:
* there is no food. only other snakes

4. Health:
* there is no health. only your snake's length

5. End conditions:
* Time is over
* Only one snake left behind
* None of snakes can move (all stuck)

**Comment by mrdrprofuroboros** [comment]
and in the original game your snake had a vision scope of 7x7 https://youtu.be/t5NMuIA4WpI?t=36
you can also implement it with api restriction, so they won't be able to see the whole field at once
